Sources/AppcuesKit/Appcues+Config.swift

```diff
@@ -194,5 +196,50 @@ public extension Appcues {
             self.enableTextScaling = enabled
             return self
         }
+
+        /// Set a custom path to font assets in the app bundle.
+        ///
+        /// Any `.ttf` files in the specified subdirectory will be registered for usage across your application.
+        ///
+        /// Usage of this configuration option is unnecessary if your custom fonts are registered in your Info.plist `UIAppFonts` as
+        /// [recommended by Apple](https://developer.apple.com/documentation/uikit/text_display_and_fonts/adding_a_custom_font_to_your_app).
+        ///
+        /// - Parameter path: Path to font files. Leading and trailing slashes are optional.
+        /// - Returns: The `Configuration` object.
+        @discardableResult
+        @objc
+        public func bundleFontsPath(_ path: String?) -> Self {
+            self.bundleFontsPath = path
+
+            if let path = path {
+                let fontResources = Bundle.main.urls(forResourcesWithExtension: "ttf", subdirectory: path) ?? []
+
+                if fontResources.isEmpty {
+                    logger.info(
+                        "No fonts found in main bundle in subdirectory %{public}s",
+                        path
+                    )
+                }
+
+                fontResources.forEach { fontURL in
+                    var error: Unmanaged<CFError>?
+
+                    if !CTFontManagerRegisterFontsForURL(fontURL as CFURL, .process, &error) {
```

Comment:
There are different scope options: https://developer.apple.com/documentation/coretext/ctfontmanagerscope

`.process` seems right from the descriptions above, but would appreciate a sanity check.

Reply:
yeah, seems like it. I don't really understand what "current use session" would mean and how that would differ from process though.